twins/service.go

```diff
@@ -205,7 +207,10 @@ func (ts *twinsService) UpdateTwin(ctx context.Context, token string, twin Twin,
 	id = twin.ID
 	b, err = json.Marshal(tw)
 
-	return nil
+	if err := ts.twinCache.Remove(ctx, id); err != nil {
```

Comment:
if we remove succeeds and save doesn't we have lost the data, right?

Reply:
Forgot to fix that :)